lib/pxclient.js

```diff
@@ -182,5 +209,25 @@ class PxClient {
             cb();
         }
     }
+
+    sendRemoteLog(message, severity, errorType, config) {
+        const enforcerConfig = new PxConfig(config, new PxLogger(config));
+        const reqHeaders = {
+            'Authorization': 'Bearer ' + enforcerConfig.config.LOGGER_AUTH_TOKEN,
+            'Content-Type': 'application/json',
+        };
+        const logParser = new PxLogsParser(config);
+        const logs = [{message, severity, errorType}];
+        logParser.enrichLogs(logs);
```

Comment:
`logParser.enrichLogs(logs);` won't have any effect, did you mean?
 `const logs = logParser.enrichLogs([{message, severity, errorType}]);`

Reply:
It has an effect it is doing Object.assign inside and log is passed by reference so it does changing the log object